addons/website/static/src/js/widgets/theme.js

```diff
@@ -373,6 +373,23 @@ var ThemeCustomizeDialog = Dialog.extend({
                         _processItems($item.children(), $element.find('.o_theme_customize_selection'), true);
                         break;
 
+                    case 'FONTSELECTION':
+                        var $options = $();
+                        _.times(parseInt(self.style.getPropertyValue('--number-of-fonts')), function (font) {
+                            var $option = $('<opt/>', {
+                                'data-widget': 'auto',
+                                'data-variable': $item.data('variable'),
+                                'data-value': font + 1,
+                                'data-font': font + 1,
+                            });
+                            $options = $options.add($option);
+                        });
+                        $element = $(core.qweb.render('website.theme_customize_dropdown_option'));
+                        var $selection = $element.find('.o_theme_customize_selection');
+                        _processItems($options, $selection, true);
+                        $selection.append($(core.qweb.render('website.theme_customize_add_google_font_option')));
```

Comment:
What is this doing in this commit? should be in the google font one.

Reply:
Yes indeed, modified this mistake